terraform/production/variables.tf

```diff
@@ -0,0 +1,33 @@
+variable "aws_region" {
+  type        = string
+  description = "region output by cloud foundry service-key command"
+  default     = "us-gov-west-1"
+}
+
+variable "cf_api_url" {
+  type        = string
+  description = "cloud.gov api url"
+  default     = "https://api.fr.cloud.gov"
+}
+
+variable "cf_org_name" {
+  type        = string
+  description = "cloud.gov organization name"
+  default     = "hhs-acf-prototyping"
```

Comment:
Because this is changing in a few days, should we go off id#? Or does this sufficiently get updated in a dynamic way and this default is less-used?